shell/gpu/gpu_surface_gl.cc

```diff
@@ -2,13 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "gpu_surface_gl.h"
```

Comment:
Is the change in line 5 necessary? I don't see the style guide of it on https://google.github.io/styleguide/cppguide.html

Reply:
Agreed, I'm not sure why I made this change back then, and I've removed it from the latet version